tuple-sketch-expansion/src/main/java/org/apache/druid/query/aggregation/datasketches/tuple/expansion/MetricFilterSelector.java

```diff
@@ -41,13 +41,13 @@ public MetricFilterSelector(
 
   public boolean matches()
   {
-    return valueMatcher.matches();
+    return valueMatcher.matches(true);
```

Comment:
@AbbyBeeler maybe you have an insight here.  `includeUnknown` - `true` / `false`?

This is the function description:
```
/**
   * Returns true if the current row matches the condition.
   *
   * @param includeUnknown mapping for Druid native two state logic system into SQL three-state logic system. If set
   *                       to true, this method should also return true if the matching result is 'unknown', such
   *                       as from the input being null valued. See {@link NullHandling#useThreeValueLogic()}.
   *
   * @return true if the current row matches the condition, or is unknown and {@code includeUnknown} is set to true
   */
```

Reply:
Configuration settings seem straight forward - I'm seeing a `public void configure(Binder binder)` method on `DruidModule` and I think you just wire it up there using `JsonConfigProvider.bind` ; 

I'm not exactly sure where the config json resides though - if it comes from environment or elsewhere - but this seems to be that pattern.

Reply:
Actually - I'm going to defer this. I don't have time to grok the intricacies right now. The part where the module binds a config class is straight forward - but I don't follow how the pattern works from there onwards.

Reply:
doesn't `includeUnknown=true` mean that unknown aka null values would be treated as always matching? I agree with Ideon that I think it should be false, but you don't need to make it configurable

Reply:
Since you already have it working locally, can you try that and test that cohorts still work? There's a chance that we don't even allow nulls so this is moot anyways

Reply:
Ok - I've switched to false. Both values give me the same results in test queries, so we'll just have to keep our eyes open!